<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>packWidget</class>
 <widget class="QWidget" name="packWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>打包</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>14</y>
     <width>256</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnPackPath">
        <property name="text">
         <string>文件路径</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineFilePath"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>项目类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboProjectType"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>设备类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboDevType"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnConfirm">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
